scraper/scraper.py

```diff
@@ -53,16 +56,16 @@ def get_scholarshipscom_details(url, appendable_url, filename):
         scholarship['name'] = row.find(
             'td', attrs={'class': 'scholtitle'}).text
         scholarship['url'] = appendable_url + str(row.a['href'])
-        scholarship['amount'] = row.find(
-            'td', attrs={'class': 'scholamt'}).text
-        scholarship['deadline'] = row.find(
-            'td', attrs={'class': 'scholdd'}).text
+        scholarship['amount'] = toAmount(row.find(
+            'td', attrs={'class': 'scholamt'}).text)
+        scholarship['deadline'] = toDate(row.find(
+            'td', attrs={'class': 'scholdd'}).text)
         scholarshipList.append(scholarship)
 
     print(scholarshipList)
```

Comment:
Can you comment this out/remove it? Makes it difficult to read error codes while machine is spinning up.